### DISCUSSION TITLE
questions about prometheus metrics data

### DISCUSSION THREAD

**Discussion by liuguangyu133** [apache/apisix#8178] [Q&A]
metrice信息里 只有apisix_http_status有域名和uri的label，(apisix_http_latency_bucket 和 apisix_bandwidth没有)
请问下这个是有什么原因吗？（比如会降低性能之类的）
按照我理解，一般来说一个域名对应一条业务线，所以我个人感觉能统计出每个域名的信息可能会更加实用(比如每个域名的带宽和p95这些信息)

![image](https://user-images.githubusercontent.com/113016850/197924701-d69b5a93-65b6-4944-a5ec-975c37f4470f.png)

**Comment by tzssangglass** [comment]
> 请问下这个是有什么原因吗？（比如会降低性能之类的）
> 按照我理解，一般来说一个域名对应一条业务线，所以我个人感觉能统计出每个域名的信息可能会更加实用(比如每个域名的带宽和p95这些信息)

now you can do it by setting `extra_labels`: https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/prometheus.md#specifying-metrics
